include/utils/grpchelper.hpp

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2024 Renesas Electronics Corporation.
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef GRPCHELPER_HPP_
+#define GRPCHELPER_HPP_
+
+#include <grpcpp/security/credentials.h>
+#include <grpcpp/security/server_credentials.h>
+
+#include <aos/common/crypto.hpp>
+#include <aos/common/cryptoutils.hpp>
+#include <aos/iam/certhandler.hpp>
+
```

Comment:
Shall we add `UtilsGRPC` namespace as for other? We have to discuss namespace usage.

Reply:
not relevant